mern-ecommerce/controllers/auth.js

```diff
@@ -73,10 +73,12 @@ exports.isAuth = (req, res, next) => {
 };
 
 exports.isAdmin = (req, res, next) => {
-  if (req.profile.role === 0) {
-    return res.status(403).json({
-      error: 'Admin resource! Access denied',
+  if (req.profile.rola == 0) { 
```

Comment:
`rola` should be `role`. It's a typo. It should be `if (req.profile.role == 0)`.